.github/workflows/README.md

```diff
@@ -0,0 +1,157 @@
+# GitHub Actions Workflows
+
+## Docker Image Management
+
+### Quick Start
+
+For your first PR with Docker changes:
+```bash
+# After pushing your PR, you'll see CI fail with "manifest unknown"
+# This is expected! Now publish the images:
+# 1. Add the "docker-image-build" label to your PR in GitHub, or
+# 2. Manually trigger the workflow from the Actions tab
+
+# Wait for docker-images workflow to complete, then re-run failed CI jobs
+```
+
+### How It Works
+
+Docker images are built automatically when Docker files change, but only published to the registry when explicitly requested via PR labels or manual triggers. This ensures version immutability and prevents accidental overwrites.
+
+#### Workflow Triggers
+
+The `docker-images.yml` workflow runs when:
+- Docker-related files change (Dockerfile, docker-compose.yml)
+- A PR is labeled with `docker-image-build`
+- Manually triggered from GitHub Actions UI
+
+#### Publishing Docker Images
+
+Images are **only** published to GitHub Container Registry when:
+1. A PR has the `docker-image-build` label
+2. You manually trigger the workflow from GitHub Actions UI
+
+When triggered by file changes alone, images are built but NOT published.
+
+#### Creating a Docker Release
+
+When you need to publish new Docker images:
+
+1. **Update the version** in `docker-compose.yml` (if needed):
+   ```yaml
+   image: ghcr.io/xwp/vip-site-template--wordpress:2.3.0  # ← Bump this
+   ```
+
+2. **Commit and push** your changes:
+   ```bash
+   git add docker-compose.yml local/docker/
+   git commit -m "Update WordPress container to Node.js 20"
+   git push
+   ```
+
+3. **Add the `docker-image-build` label** to your PR:
+   - Go to your PR on GitHub
+   - Click "Labels" in the right sidebar
+   - Add the `docker-image-build` label
+
+4. The `docker-images.yml` workflow will:
+   - Build the images
+   - Push them to `ghcr.io` with the version from docker-compose.yml
+   - Make them available for all future CI runs
+
+
```

Comment:
Not a big deal, but have an extra line break here. If you have other changes maybe remove one of the line breaks.